### ISSUE TITLE
IllegalArgumentException at runtime when try to wire an actor with an implicit parameter

### ISSUE THREAD

**Issue by akkie** [adamw/macwire#121]
I currently try to wire an actor with an implicit parameter. The project compiles but I get an IllegalArgumentException at runtime.

The constructor looks as follow:

```scala
class Connector(brokerService: BrokerService)(
  implicit
  ec: ExecutionContext
) extends Actor with LazyLogging
```
The error is the follow:
```
java.lang.IllegalArgumentException: no matching constructor found on class app.actors.Connector for arguments [class app.models.services.BrokerServiceImpl]
```

 I instantiate the actor with `val connector: ActorRef = wireActor[Connector]("connector")`. The `BrokerService` is injected in the class which instantiates the connector actor.

Reference: https://gitter.im/adamw/macwire?at=59db4087bac826f054507d2e

**Comment by fredfp** [comment]
@akkie could you try my fix for this issue?

**Comment by adamw** [comment]
Released in 2.3.1
